website/docs/components/text/partials/code/how-to-use.md

```diff
@@ -136,5 +154,4 @@ We can imagine even more complex examples where we may want to apply a typograph
 While this is technically possible, it’s not necessarily desirable (or correct in every possible context).
 Please be mindful of how the markup is structured, in terms of HTML semantics (for accessibility) as well as code readability (for the developers writing/reviewing/maintaining that code, now and in the future).
 
-!!!
-
+!!! -->
```

Comment:
[blocker] this is a closing comment tag, but why is it here?

Reply:
Good catch, I removed it. I was running into a bug and testing some things.